protos/sigstore_trustroot.proto

```diff
@@ -117,3 +118,50 @@ message TrustedRoot {
         // A set of trusted timestamping authorities.
         repeated CertificateAuthority timestamp_authorities = 5;
 }
+
+
+// SigningConfig represents the trusted entities/state needed by Sigstore
+// signing. In particular, it primarily contains service URLs that a Sigstore
+// signer may need to connect to for the online aspects of signing.
+message SigningConfig {
+        // A URL to a Fulcio-compatible CA, capable of receiving
+        // Certificate Signing Requests (CSRs) and responding with
+        // issued certificates.
+        //
+        // This URL **MUST** be the "base" URL for the CA, which clients
+        // should construct an appropriate CSR endpoint on top of.
+        // For example, if `ca_url` is `https://example.com/ca`, then
+        // the client **MAY** construct the CSR endpoint as
+        // `https://example.com/ca/api/v2/signingCert`.
+        string ca_url = 1;
+
+        // A URL to an OpenID Connect identity provider.
+        //
+        // This URL **MUST** be the "base" URL for the OIDC IdP, which clients
+        // should perform well-known OpenID Connect discovery against.
+        string oidc_url = 2;
+
+        // A URL to a Rekor-compatible transparency log.
+        //
+        // This URL **MUST** be the "base" URL for the transparency log,
+        // which clients should construct appropriate API endpoints on top of.
+        string tlog_url = 3;
```

Comment:
Should we support multiple URLs for multiple log entries and timestamps? [`VerificationMaterial`](https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_bundle.proto#L59) supports this already. Hopefully down the road, we'll see additional logs spun up, similar to the Certificate Transparency ecosystem, where users can submit entries to multiple logs as a mitigation against an individual log presenting a split-view.

We could say YAGNI, but it might be easier to do this now rather than have to have a breaking change later. And it shouldn't be much work for clients who ingest this message to request timestamps and log proofs for each URL, adding each to the respective repeated bundle field.

Reply:
Yeah, that makes sense to me! I'll make those changes.

Reply:
[1478df1](https://github.com/sigstore/protobuf-specs/pull/277/commits/1478df19e31c3c26967ef8c549c475e7b5fe7301)